core/include/drivers/gic.h

```diff
@@ -56,4 +58,7 @@ void gic_init_per_cpu(void);
 
 /* Print GIC state to console */
 void gic_dump_state(void);
+
+/* Release the property for a shared peripheral interrupt */
```

Comment:
Explicitly say that the interrupt is re-assigned to the non-secure interrupt group and its priority reset to `GIC_SPI_PRI_NS_EL1`.
State also that the interrupt shall be disabled when this function is called.

Reply:
done